lib/swaps/Swaps.ts

```diff
@@ -677,6 +676,53 @@ class Swaps extends EventEmitter {
     return true;
   }
 
+  private handleHtlcAccepted = async (swapClient: SwapClient, rHash: string, amount: number, currency: string) => {
```

Comment:
I will make a follow-up issue for this one since the PR here is only breaking it into its own named function and I don't want this to get too much larger.